### PR TITLE
REAC-327 Add api docs for grid filtering plugin

### PR DESCRIPTION
**Pull request by tommymorgan**

### CODE REVIEW

**Comment by dalley** [comment]
@mgusmano @jnsencha please review.  This is updated documentation.

**Comment by dalley** [comment]
@branflake2267 can you review?

File: api-docs/modern/modern/src/grid/filters/Plugin.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
+/**
+ * This plugin enables user-defined filters for a grid.
+ * @since 6.7.0
+ *
+ * In general an gridfilters plugin will be passed to the grid:
+ *
+ *     @example
```
Comment (branflake2267) [inline api-docs/modern/modern/src/grid/filters/Plugin.js]: @example is missing the extreact package. Could you add the package?

**Comment by branflake2267** [comment]
@dalley One missing item. The example looks great other than that!

**Comment by dalley** [comment]
@branflake2267 @jnsencha I've updated the code looks like the package was missing from a few of the other ones that were merged already.  I have fixed them all.

**Pull request merged**